index.html

```diff
@@ -181,7 +200,7 @@ <h4>LET'S CONNECT</h4>
                 </ul>
                 </nav>
         </section>
-        <footer class="contact">
+        <footer class="contact" id="contact">
```

Comment:
- [ ] I think you kind of introduced an overflow in your contact section. Please adjust it to match the Figma design template chosen 👍🏽 

Your Design 👇🏽 

![image](https://user-images.githubusercontent.com/63473584/188228144-267afc07-41df-4f35-a88f-75b9e0c21d57.png)

Figma Design 👇🏽 

![image](https://user-images.githubusercontent.com/63473584/188228352-def84059-3987-4e4b-bb61-cc494c568791.png)